themes/default/content/learn/pulumi-fundamentals/configure-and-provision/index.md

```diff
@@ -317,6 +316,20 @@ network = docker.Network("network", name=f"services-{stack}")
 
 {{% choosable language java %}}
 
+Add these imports to the top:
```

Comment:
Given the last line in the paragraph above this one ("add this code to the bottom"), this doesn't quite read right:

![image](https://user-images.githubusercontent.com/274700/166997860-8beec6f4-2b4a-4e16-86d2-ecfbfd8111d9.png)

```suggestion
Add these imports to the top:
```

If it were me, I'd probably adjust the preceding paragraph to be less explicit about "where" -- i.e., remove "at the bottom of", so it's just "add the following code to your program". The non-Java cases _should_ be fine, I think, since "add" implies "to the bottom", but if you wanted to use a comment (`// ...`) to indicate positioning, that might help, too. Up to you, though, as I know you want to get this merged.

Reply:
I think we can follow this up later tonight; I'm running out of time to get this done in time for it to be live by workshop time!

Reply:
Totally, sounds good to me!